lib/utils/index.js

```diff
@@ -341,17 +341,37 @@ function hasChangesInForbiddenSections(diff) {
     .map(line => _.nth(/^([a-z]\w+):/.exec(line), 1))
     .difference([
       'update',
-      'compilation',
       'releases',
-      'resource_pools',
-      'networks',
-      'properties',
       'tags',
       'addons'
     ])
     .value();
+
+  function isJobRemoved() {
+    const jobsRegex = new RegExp('^  jobs'); // this regex is to find the position of jobs section
+    const jobsLevelRegex = new RegExp('^  [a-z]+'); // this regex is to find the position of next section at the same level as jobs
+    const jobNameRegex = new RegExp('^  [- ] name'); // this regex is to find the position of removal in job name
+    const jobStartIndex = _.findIndex(diff, element => {
+      return jobsRegex.test(element[0]);
+    });
+    const jobEndIndex = _.findIndex(diff, element => {
```

Comment:
similarly above can be written like below. If the method has only one statement and return, then follow the below concise way.
```
const jobEndIndex = _.findIndex(diff, jobsLevelRegex.test(element[0]), jobStartIndex + 1);
```

Reply:
ok